integreat_cms/cms/views/dashboard/dashboard_view.py

```diff
@@ -131,25 +134,28 @@ def get_unread_feedback_context(self) -> dict[str, QuerySet]:
         :return: Dictionary containing the context for unreviewed pages
         """
         unread_feedback = Feedback.objects.filter(
-            read_by=None, archived=False, region=self.request.region
+            read_by=None,
+            archived=False,
+            region=self.request.region,
         )
         return {
             "unread_feedback": unread_feedback,
         }
 
     @json_response
     def get_broken_links_context(
-        # pylint: disable=no-self-argument
-        request: HttpRequest,
-        region_slug: str,  # pylint: disable=unused-argument
+        request: HttpRequest,  # noqa: N805
+        **kwargs: Any,
     ) -> JsonResponse:
```

Comment:
Does this have to be an instance method? What about marking it with `@staticmethod`? The only place I can find it being used is in `urls/protected.py` and it has no connection to an instance or the class.

Reply:
No idea, will check.